test/Libraries/Microsoft.Extensions.Caching.Hybrid.Tests/Microsoft.Extensions.Caching.Hybrid.Tests.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreTargetFrameworks)</TargetFrameworks>
```

Comment:
Any specific reason not to test net462 version?

Reply:
I've added net462, but I'm having difficulty validating that locally - the runner seems to have a SNK problem - and I expect it to fail the CI build:

``` txt
[xUnit.net 00:00:00.72] Microsoft.Extensions.Caching.Hybrid.Tests: Catastrophic failure: System.IO.FileLoadException: Could not load file or assembly 'Microsoft.Extensions.Caching.Hybrid.Tests, Version=9.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. Strong name signature could not be verified.  The assembly may have been tampered with, or it was delay signed but not fully signed with the correct private key. (Exception from HRESULT: 0x80131045)
File name: 'Microsoft.Extensions.Caching.Hybrid.Tests, Version=9.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
```

If I remove the signing, I get instead:

``` txt
Friend access was granted by 'Microsoft.Extensions.Caching.Hybrid, Version=9.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35', but the public key of the output assembly ('') does not match that specified by the InternalsVisibleTo attribute in the granting assembly.
```

Any advice on making that happier?

Reply:
It looks like your issue may have gone away. The CI build is green, and I tried your branch locally - al tests are green.
![image](https://github.com/user-attachments/assets/cc585ce8-0c14-49d6-b8b1-24c816ff2bcf)